src/ArduinoGraphics.cpp

```diff
@@ -184,6 +184,95 @@ void ArduinoGraphics::rect(int x, int y, int width, int height)
   }
 }
 
+void ArduinoGraphics::ellipse(int x, int y, int width, int height)
+{
+  if (!_stroke && !_fill) {
+    return;
+  }
+
+  int x1 = x;
+  int y1 = y;
+  int r1 = (int)(width/2);
+  int r2 = (int)(height/2);
+  int x2 = x1 + r1 - 1;
+
+  for (x = x1; x <= x2; x++) {
+      y2 = (int) (y1 + sqrt((1 - ((x * x)/(r1 * r1)))) * r2);
```

Comment:
```suggestion
      int y2 = (int) (y1 + sqrt((1 - ((x * x)/(r1 * r1)))) * r2);
```
Lack of variable declaration causes compilation to fail.